draft-ietf-masque-h3-datagram.md

```diff
@@ -76,46 +78,48 @@ document are to be interpreted as described in BCP 14 {{!RFC2119}} {{!RFC8174}}
 when, and only when, they appear in all capitals, as shown here.
 
 
-# Flow Identifiers {#flow-id}
+# Flows
```

Comment:
Do we want to call them "flows" or "datagram flows"? The second might be slightly clearer, albeit more wordy.

Reply:
I thought the same initially, I could go either way because the wordings doesn't bother me.

Reply:
Maybe we go with datagram flow then since it's slightly clearer and matches the header name?

Reply:
agreed, done